projects/plugins/jetpack/extensions/blocks/writing-prompts/editor.js

```diff
@@ -0,0 +1,27 @@
+import { createBlock } from '@wordpress/blocks';
+import { dispatch } from '@wordpress/data';
+import { waitForEditor } from '../../shared/wait-for-editor';
+
+async function insertTemplate( prompt, embedPrompt = false ) {
+	await waitForEditor();
+
+	const { insertBlocks } = dispatch( 'core/block-editor' );
+	const writingPromptBlocks = embedPrompt
+		? [ createBlock( 'core/pullquote', { value: prompt.text } ), createBlock( 'core/paragraph' ) ]
```

Comment:
in fact, it looks (to me) like the app inserts the `prompt.content` which includes markup for a pullquote. That is another option, I actually think it  probably makes more sense to always use prompt.text for consistency though, what do you think?

Reply:
I prefer telling the client to insert the relevant block(s).

I think sending block markup generated from one editor at a specific point in time in the past to another editor seems less robust then just telling the editor to insert a pullquote block with the specified content.